packages/react/test/specs/components/Animation/Animation-test.tsx

```diff
@@ -2,11 +2,22 @@ import * as React from 'react'
 import { isConformant } from 'test/specs/commonTests'
 
 import Animation from 'src/components/Animation/Animation'
+import { consoleUtil, mountWithProvider } from 'test/utils'
 
 describe('Animation', () => {
   isConformant(Animation, {
     hasAccessibilityProp: false,
     requiredProps: { children: <div /> },
     handlesAsProp: false,
   })
+
+  test('does not throw if children is not passed', () => {
+    consoleUtil.disableOnce()
+    // @ts-ignore
+    expect(() => mountWithProvider(<Animation />)).not.toThrowError()
+
+    consoleUtil.disableOnce()
+    // @ts-ignore
```

Comment:
Ouch, why? 😭

Reply:
We are testing if user, passes undefined it will not throw, and the typings are not allowing that, so we must use ts-ignore (feels awkward, but I don't have a better idea of how to test it...)

Reply:
@miroslavstastny any better idea?

Reply:
no